lib/assign-properties.js

```diff
@@ -37,7 +45,7 @@ function giveMethodSuper(superclass, name, fn) {
 
 var sourceAvailable = (function() {
   return this;
-}).toString().indexOf('return this;') > -1;
+}).toString().indexOf('return this') > -1;
```

Comment:
can you submit this separately, this is pretty obvious good thing to get in asap. I would like to review the rest in more detail